### PR TITLE
Bump Barstools for Deprecations Fixes

### PR DESCRIPTION
**Pull request by chick**
**Related Issues/PRs:** https://github.com/ucb-bar/barstools/pull/98 https://github.com/ucb-bar/barstools/pull/97

**Type of change**: removing deprecated code from barstools

**Impact**: No impact except less warnings

**Release Notes**
Removed deprecated Driver code from barstools

### CODE REVIEW

**Comment by colinschmidt** [comment]
We should have someone who is actively using hammer also test with this branch ideally. I think the barstools code paths for that are different than the CI.

**Comment by jerryz123** [comment]
I'll try.

File: .circleci/config.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ parameters:
 executors:
     main-env:
         docker:
-            - image: ucbbar/chipyard-image:1.0.1
+            - image: ucbbar/chipyard-ci-image:84cfc73
```
Comment (abejgonzalez) [inline .circleci/config.yml:15]: For this PR, can you revert this change. Just modify the `timeout` settings and not this image.

Comment (chick) [reply .circleci/config.yml]: Reverted docker change, and re-changed lines 115 and 234 to use 20m timeout

**Comment by jerryz123** [comment]
The macrocompiler flow still seems to work fine.

**Review by jerryz123 — APPROVED** [review]
Approved, pending the change Abe requested.

**Review by abejgonzalez — APPROVED** [review]
LGTM pending CI
